lib/background_worker.rb

```diff
@@ -25,7 +25,7 @@ def self.logger
 
   def self.verify_active_connections!
     Rails.cache.reconnect if defined?(Rails)
-    ActiveRecord::Base.verify_active_connections! if defined?(ActiveRecord)
+    ActiveRecord::Base.connection_pool.connections.each {|c| c.verify! } if defined?(ActiveRecord)
```

Comment:
I think we actually need to use the clear_active_connections! method, as the old method would also release stale connections from the pool